src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -37,8 +38,7 @@ internal TagHelperExecutionContext(string tagName, bool selfClosing)
         /// Instantiates a new <see cref="TagHelperExecutionContext"/>.
         /// </summary>
         /// <param name="tagName">The HTML tag name in the Razor source.</param>
-        /// <param name="selfClosing">
-        /// <see cref="bool"/> indicating whether or not the tag in the Razor source was self-closing.</param>
+        /// <param name="tagMode">A mode indicating how the tag was written in the Razor source.</param>
```

Comment:
For this and the associated property here and through classes such as `TagMode` (though that one needs to be ambiguous w.r.t. input and output) and so on, suggest `HTML syntax of the element in the Razor source.`